projects/plugins/boost/app/modules/optimizations/page-cache/pre-wordpress/Request.php

```diff
@@ -53,7 +53,7 @@ public function get_uri() {
 	}
 
 	public function get_parameters() {
-		return $this->request_parameters;
+		return apply_filters( 'jetpack_boost_cache_parameters', $this->request_parameters );
```

Comment:
Can you add some docblock to document this filter?

Reply:
Will do.